src/fhirtypes/ElementDefinition.ts

```diff
@@ -2662,7 +2633,7 @@ const PROPS = [
   'mapping'
 ];
 
-const CHOICE_TYPE_SLICENAME_POSTFIXES = [
+export const CHOICE_TYPE_SLICENAME_POSTFIXES = [
```

Comment:
Given that this constant is only used by `getShortcutId`, I think it makes sense to move this to `common.ts` too.  That way it also doesn't need to be exported (since no one else uses it).